### ISSUE TITLE
IP Address Not Found Error

### ISSUE THREAD

**Issue by pabz616** [RhinoSecurityLabs/cloudgoat#181]
**Description**
At the step where I run the config option to whitelist my IP, the IP address is not getting found, whitelist.txt is not getting created 

**Commit**
[e0338d9](https://github.com/RhinoSecurityLabs/cloudgoat/commit/e0338d9cd6f66206d0d053b3ee50635ab23ec4f9)

**System**
Mac M1, running Ventura 13.3

**Steps**
1. Completed initial steps of installing CloudGoat on my machine
2. Completed docker setup
3. Cloned the repo and completed all the steps up running 
`$ ./cloudgoat.py config profile`

**Expected Result**
* whitelist.txt is created with my IP address added

**Actual Result**
* I ran the following config step and got this error
`$ ./cloudgoat.py config whitelist --auto`
<img width="863" alt="no IP" src="https://user-images.githubusercontent.com/22268013/229907583-b304ff6d-5d49-4265-b968-776f34a1b244.png">

**Workaround**
* I retried these steps in my Kali Linux and encountered the same steps

**Comment by DaveYesland** [comment]
Do you get an error if you just run:
`curl https://ifconfig.co/json`

or
```
# python
>>import requests
>>requests.get("https://ifconfig.co/json")
```
Just wondering if there is not something else blocking the request to https://ifconfig.co/json

**Comment by pabz616** [comment]
@DaveYesland 
1. When I curl ifconfig.co/json, I get an html response
2. With the python command in my terminal, I'm getting a 403

**Comment by pabz616** [comment]
No

**Comment by pabz616** [comment]
@DaveYesland - Is there a boilerplate to whitelist.txt? I'm going to create my own and set my IP that way.

**Comment by DaveYesland** [comment]
You should be able to just create the whitelist.txt containing just the IP as a CIDR or you can use the `config whitelist` command and manually set it.
```
    config whitelist [--auto]
        CloudGoat needs to know what IP addresses should be
        whitelisted when potentially-vulnerable resources are
        deployed, and these IPs are tracked as CIDR ranges in a
        whitelist.txt file in CloudGoat's base directory. You may
        create and fill in this file manually, or this command may
        be used to walk through the creation process. If a whitelist
        file already exists and contains valid IP addresses, it will
        display them.
```

**Comment by pabz616** [comment]
@DaveYesland I think I got some good news! 

I managed to correct the IP issue in the following manner:
1. ifconfig was giving me IPV6 IP address which cloudgoat wasn't having, so I went to whatismyip.com to get the IP (IPV4)
2. Then I visited this site - https://www.meridianoutpost.com/resources/etools/network/convert-ip-to-CIDR.php? - to convert my IP to CIDR
3. I kicked off the docker instance to get to the directory `usr/src/cloudgoat#` and I created the whitelist.txt file there

Now I got a different error, but this ticket can be closed
